scripts/fzf_python.md

```diff
@@ -0,0 +1 @@
+echo '' | fzf --print-query --preview 'python -c {q}'
```

Comment:
Could you please rename this file to `fzf_python.sh`?

Reply:
Done